manifests/0000_90_console-operator_02_servicemonitor.yaml

```diff
@@ -0,0 +1,42 @@
+# Configure cluster-monitoring for console-operator
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: console-operator
+  namespace: openshift-console-operator
+spec:
+  endpoints:
+    - bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
+      interval: 30s
+      path: /metrics
+      port: https
+      scheme: https
+      tlsConfig:
+        caFile: /etc/prometheus/configmaps/serving-certs-ca-bundle/service-ca.crt
+        serverName: metrics.openshift-console-operator.svc
+  jobLabel: component
+  selector:
+    matchLabels:
+      name: console-operator
+---
+# Configure cluster-monitoring for cluster console resources
```

Comment:
If we're not yet exposing a metrics endpoint for console, should we remove this?

Reply:
The `openshift-console` namespace will be ignored until we add the label here https://github.com/openshift/console-operator/pull/270/files#diff-7a267258d7275352227e1bee37a51bfaR9.

So I think it's ok we leave the rest of this in place.  It's ready to go when we want to turn it on.

Reply:
@benjaminapetersen We should only add this when we plan to use it. Particularly because it won't be until 4.3 at least.

Reply:
can do.